test/rubygems/test_gem_uninstaller.rb

```diff
@@ -221,13 +221,13 @@ def test_uninstall_default_gem
 
     uninstaller = Gem::Uninstaller.new spec.name, :executables => true
 
-    e = assert_raises Gem::InstallError do
+    use_ui @ui do
       uninstaller.uninstall
     end
 
-    assert_equal 'gem "default" cannot be uninstalled ' +
-                 'because it is a default gem',
-                 e.message
+    lines = @ui.output.split("\n")
+
+    assert_equal 'Ignored default-2 because it is a default gem', lines.shift
```

Comment:
What is wrong with the original error message?

Reply:
You're right, the previous message was probably as good as this one. I'll revert to the one in place.

Reply:
@bronzdoc updated!